src/screens/RegisterAccount.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import {
+  SafeAreaView,
+  View,
+  ScrollView,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+import {Colors} from '../utils/Colors';
+
+const RegisterAccount = () => {
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView>
+        <Text style={styles.title}>Crea tu cuenta</Text>
+        <View style={styles.formContainer}>
+          <View style={styles.inputsContainer}>
+            <View style={styles.textContainer}>
+              <Text style={styles.titleText}>Correo electrónico</Text>
+              <Text style={[styles.titleText, styles.requiredText]}>
+                Requerido
+              </Text>
+            </View>
+            <TextInput style={styles.textInput} />
+          </View>
+          <View style={styles.inputsContainer}>
+            <View style={styles.textContainer}>
+              <Text style={styles.titleText}>Clave</Text>
+              <Text style={[styles.titleText, styles.requiredText]}>
+                Requerido
+              </Text>
+            </View>
+            <TextInput style={styles.textInput} secureTextEntry={true} />
+          </View>
+          <View style={styles.requirementsContainer}>
+            <Text style={styles.textInfo}>La clave debe:</Text>
+            <View style={styles.unorderedListContainer}>
+              <Text style={styles.textInfo}>
+                <Text style={styles.dash}>-</Text> tener 8 o más caracteres
+              </Text>
+              <Text style={styles.textInfo}>
+                <Text style={styles.dash}>-</Text> contener por lo menos 1 letra
+                minúscula
+              </Text>
+              <Text style={styles.textInfo}>
+                <Text style={styles.dash}>-</Text> contener por lo menos 1 letra
+                mayúscula
+              </Text>
+              <Text style={styles.textInfo}>
+                <Text style={styles.dash}>-</Text> contener por lo menos 1
+                dígito
+              </Text>
+            </View>
+          </View>
+          <View style={styles.inputsContainer}>
+            <View style={styles.textContainer}>
+              <Text style={styles.titleText}>Escribe de nuevo la clave</Text>
+              <Text style={[styles.titleText, styles.requiredText]}>
+                Requerido
+              </Text>
+            </View>
+            <TextInput style={styles.textInput} secureTextEntry={true} />
+          </View>
+          <TouchableOpacity
+            style={styles.nextBtn}
+            onPress={() => {
+              console.log('Hello world');
+            }}>
+            <Text style={styles.nextBtnText}>Siguiente</Text>
```

Comment:
could you create a file with the constant for all the text?
like export const NEXT_MESSAGE_ES = 'Siguiente';